src/adapter/src/catalog.rs

```diff
@@ -4583,6 +4590,8 @@ impl From<ConcreteComputeInstanceReplicaConfig> for SerializedComputeInstanceRep
 pub enum SerializedComputeInstanceReplicaLocation {
     Remote {
         addrs: BTreeSet<String>,
+        compute_addrs: BTreeSet<String>,
```

Comment:
This is persisted to the stash, so changing it requires either a migration or to wait for a wipe release to commit this PR. (We're aware we don't have automated tests for detecting this, it's on our priority list somewhere.)

I thiiink our current main is already a breaking release, so it might be safe to merge this already? You should check with, um, the #release channel first before merging though.